configure

```diff
@@ -1563,6 +1565,10 @@ for ac_option in $@; do
 		_pandocformat="$arg"
 		_pandoc=yes
 		;;
+	--with-manual-version=*)
```

Comment:
what about adding a sphinx check to let user decide if he prefers to build or download the PDF manual?

Reply:
It's not enough to check for sphinx, it's a full python env with all the necessary sub-dependencies. What's the benefit of build it all locally vs just pulling the version?

Reply:
It's about not depending on an external service to build the whole application.

Reply:
I thought about this - realistically the Manual only needs to be built when we package for distribution, so there's no impact when building the application. We really only care about this for the few platforms that have a PDF viewer - PC, Mac, iOS, Android - but the preference should always be to redirect users to the docs site if possible.

Reply:
That makes sense. Thanks